src/components/components/TxTemplate.re

```diff
@@ -10,73 +20,58 @@ let make =
         <WildcardsLoader />
       </Rimble.Heading>
       <Rimble.Text> "Tx Created."->React.string </Rimble.Text>
-      <Rimble.Loader
-        className=Styles.centerItems /*color="green"*/
-        size="80px"
-      />
+      <Rimble.Loader className=Styles.centerItems size="80px" />
     </React.Fragment>
-  | AnimalActions.SignedAndSubmitted =>
+  // TODO: make it show the link to the tx on etherscan.io!
+  | AnimalActions.SignedAndSubmitted(txHash) =>
     <React.Fragment>
       <Rimble.Heading>
         "Processing Transaction "->React.string
         <WildcardsLoader />
       </Rimble.Heading>
-      <Rimble.Text> "Tx ."->React.string </Rimble.Text>
-      <Rimble.Loader
-        className=Styles.centerItems /*color="green"*/
-        size="80px"
-      />
+      <Rimble.Text>
+        <a href={j|https://$etherscanUrl/tx/$txHash|j} target="_blank">
+          "View the transaction on etherscan.io"->React.string
+        </a>
+      </Rimble.Text>
+      <Rimble.Loader className=Styles.centerItems size="80px" />
     </React.Fragment>
-  | AnimalActions.Complete(_result) =>
+  | AnimalActions.Complete(result) =>
```

Comment:
On complete - maybe background colour scheme or something to change when the user now owns the animal. Another suggestion Jason just mentioned was a react component that renders on success saying the animal you bought, how much you bought it for, how much you set the selling price at, how long your deposit will last etc, inline